test/jdk/com/sun/jdi/ClassUnloadEventTest.java

```diff
@@ -65,18 +63,13 @@ public static void main(String[] args) throws Exception {
         }
     }
 
-    private static class TestClassLoader extends ClassLoader implements Opcodes {
+    private static class TestClassLoader extends ClassLoader {
         private static byte[] generateSampleClass(String name) {
-            ClassWriter cw = new ClassWriter(0);
-
-            cw.visit(52, ACC_SUPER | ACC_PUBLIC, name, null, "java/lang/Object", null);
```

Comment:
This anticipates an internal name but gets a binary name passed. This works in the old code accidentally because its classes were in the unnamed package.  The port fixes this bug.